### DISCUSSION TITLE
Restricted Github login - get token manually?

### DISCUSSION THREAD

**Discussion by omachala** [community/community#40820] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

Hi, my company's network has restrictions on Github login and therefore I cannot login from my VS Code to get a token. Is there a way to generate token manually and paste in VScode? or any other workaround?

**Comment by iamcjwilliams** [comment]
Hello, @omachala Currently there's no way to generate a manual token. Unfortunately, self-assigned certificates are something copilot currently doesn't support
